Packages/com.unity.inputsystem/Documentation~/Actions.md

```diff
@@ -295,13 +295,39 @@ Instead of using callbacks, it might be simpler sometimes to poll the value of a
         moveAction.Enable();
     }
 
-    void OnUpdate()
+    void Update()
     {
         var moveDirection = moveAction.ReadValue<Vector2>();
         position += moveDirection * moveSpeed * Time.deltaTime;
     }
 ```
 
+For button-type actions, you can also use [`InputAction.triggered`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_triggered) which will true if at any time in the current frame, the action was performed.
```

Comment:
"which will true"? which will _be_ true?

Reply:
if at any time in the current frame, the action was performed. -> 
if the action was performed at any time in the current frame.

Reply:
Fixed